packages/reports/addon/components/pick-range-container.js

```diff
@@ -103,16 +103,29 @@ export default PickObjectContainer.extend({
       this.send('stagePropertyChange', '_start', start);
     },
 
+    /**
+     * Used to set the end of the advanced range selector
+     * @param {String} - date entered into box, could be date or macro.
+     */
+    setAdvancedEnd(end) {
```

Comment:
Can we call this `setExclusiveEnd` unlike the other which is inclusive. I think it will make more sense why we have one start setter for both the date picker the and the adv input and two different individual end setters for the date picker and the adv input.

Reply:
I think I can manage that.